is_core/utils/__init__.py

```diff
@@ -319,3 +359,18 @@ def get_link_or_none(pattern_name, request, view_kwargs=None):
         return pattern.get_url_string(request, view_kwargs=view_kwargs)
     else:
         return None
+
+
+class GetMethodFieldMixin:
```

Comment:
helper dělá to samé co pyston, vrátí svou metodu dle jména. Výhoda je, že to jde přetížit a vracet metody i z core nebo jakkoli jinak rozšířit